examples/Jenkinsfile.push-events

```diff
@@ -23,14 +16,22 @@ pipeline {
                 script {
                     def status = event.pushDynatraceEvent (
                       eventType: "CUSTOM_DEPLOYMENT",
-                      tagRule: tagMatchRules,
-                      deploymentName: "CUSTOM_DEPLOYMENT: ${env.JOB_NAME}",
-                      deploymentVersion: "myDeploymentVersion",
-                      deploymentProject: "myDeploymentProject",
-                      remediationAction: "myRemediationAction",
+                      title: "New Deployment: ${env.JOB_NAME}",
+                      entitySelector: selector,
                       customProperties : [
                           "Jenkins JOB_NAME": "${env.JOB_NAME}",
                           "Jenkins BUILD_NUMBER": "${env.BUILD_NUMBER}"
+                          "dt.event.deployment.name":"${env.JOB_NAME}",
+                          "dt.event.deployment.version": "1.1",
+                          "dt.event.deployment.release_stage": "production" ,
+                          "dt.event.deployment.release_product": "frontend",
+                          "dt.event.deployment.release_build_version": "123",
+                          "approver": "Jason Miller",
+                          "dt.event.deployment.ci_back_link": "https://pipelines/easytravel/123",
+                          "gitcommit": "e5a6baac7eb",
```

Comment:
put placeholders instead of real values i.e gitcommit should be  something like "gitcommit":"your-git-commit-id".
the same for the approver field .